presto-docs/src/main/sphinx/connector/hive.rst

```diff
@@ -341,6 +344,56 @@ the ``org.apache.hadoop.conf.Configurable`` interface from the Hadoop Java API,
 will be passed in after the object instance is created and before it is asked to provision or retrieve any
 encryption keys.
 
+S3SelectPushdown
+^^^^^^^^^^^^^^^^
+S3SelectPushdown is a feature of Amazon S3 Select that enables Presto users to push down projection (SELECT)
+and predicate (WHERE) processing to S3 Select. This feature enables users to retrieve only the data from
+Amazon S3 that they want to query with Presto, instead of retrieving full objects. S3SelectPushdown thus reduces
+both latency and required bandwidth when running Presto queries against Amazon S3 objects.
+
+Is S3 Select Right For My Application?
+######################################
+S3SelectPushdown's performance is dependent on the amount of data filtered by the query. Larger filtration results in better
+performance. The inverse is also true. If the query doesn't filter any data then the pushdown may not add any additional value and
+user will be charged for S3Select requests. Thus, we recommend that you benchmark your applications with and without S3 Select
+to see if using it may be suitable for your application. By default, S3SelectPushdown is disabled and you should enable it in
+production after performing suitable performance benchmarking and cost analysis.
```

Comment:
`after proper benchmarking and cost analysis`

Reply:
Fixed